src/test/regress/sql/pg17.sql

```diff
@@ -274,6 +274,34 @@ RESET ROLE;
 
 -- End of MAINTAIN privilege tests
 
+-- Correlated sublinks are now supported as of PostgreSQL 17, resolving issue #4470.
+-- Enable DEBUG-level logging to capture detailed execution plans
+SET client_min_messages TO DEBUG1;
```

Comment:
Let's reset log_remote_commands before running this test
```suggestion
RESET citus.log_remote_commands;
SET client_min_messages TO DEBUG1;
```